R/TuneMultiCritControlMBO.R

```diff
@@ -0,0 +1,67 @@
+#' @export
+#' @param learner [\code{\link{Learner}} | \code{NULL}]\cr
```

Comment:
These are the same as in TuneControlMBO, right. We should avoid copy & paste. Try to use `@inheritParams` https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html#do-repeat-yourself